spyder/widgets/pathmanager.py

```diff
@@ -71,9 +71,17 @@ def __init__(self, parent, path=None, read_only_path=None,
         self.setWindowTitle(_("PYTHONPATH manager"))
         self.setWindowIcon(ima.icon('pythonpath'))
         self.resize(500, 300)
-        self.sync_button.setVisible(os.name == 'nt' and sync)
+        self.import_button.setVisible(sync)
+        self.export_button.setVisible(os.name == 'nt' and sync)
 
         # Layouts
+        description = QLabel(
+            _("Only paths listed here will be passed to IPython consoles and "
+              "the language server<br>as additional Python search paths."
+              "<br>"
+              "Paths in your system <b>PYTHONPATH</b> environment variable "
+              "can be imported to this list.")

```

Comment:
I tried to make the wording a little clearer and more understandable, though I'm still not 100% happy with it. I also elided the hard breaks for now, since they won't line up the same as before, and may not produce the desired results at different window widths, UI scaling, font sizes and translations; if you want the second break, it should generally be a full double paragraph `<br>`.

```suggestion
            _("The paths listed below will be passed to IPython consoles and "
              "the language server as additional locations to search for "
              "Python modules. "
              "Any paths in your system <b>PYTHONPATH</b> environment "
              "variable can be imported here if you'd like to use them.")
```